doc/source/tutorial/stats/continuous_genhyperbolic.rst

```diff
@@ -0,0 +1,136 @@
+
+.. _continuous-genhyperbolic:
+
+Generalized Hyperbolic Distribution
+======================================
+
+The Generalized Hyperbolic Distribution is defined as the normal variance-mean mixture with Generalized Inverse Gaussian distribution as mixing distribution.
+The "hyperbolic" characterization refers to the fact that shape the log-probability distribution can be described as an hyperbola. Hyperbolic distributions are sometime referred as semi-fat tail because their probability density decrease slower than "sub-hyperbolic" distributions (e.g. normal distribution, whose log-probability decreases quadratically), but faster than other "extreme value" distributions (e.g. `pareto` distribution, whose log-probability decreases logarithmically).
+
+
+Functions
+---------
+
+Different parameterization exists in literature, scipy implementing the location-scale as for Prause (1999)
+
+.. math::
+
+   \begin{eqnarray*} f(x, p, \hat{\alpha}, \hat{\beta}, \text{location}, \text{scale}) &=&
+            \frac{(\hat{\alpha}^2 - \hat{\beta}^2)^{\lambda/2}}
+            {\sqrt{2\pi}\hat{\alpha}^{\lambda-0.5}
+            K_\lambda\Big(\sqrt{\hat{\alpha}^2 - \hat{\beta}^2}\Big)}
+            e^{\hat{\beta}(x)} \times \frac{K_{\lambda - 1/2}
+            (\hat{\alpha} \sqrt{1 + x^2})}
+            {(\sqrt{1 + x^2})^{1/2 - \lambda}} \end{eqnarray*}
+
+for:
+
+- :math:`x, \lambda \in \mathbb{R}`
+- :math:`|\hat{\beta}| \lt \hat{\alpha}` if :math:`p \ge 0`
```

Comment:
Did you intend to use `p` instead of `lambda` above?

Reply:
probably yes, i will address this together with https://github.com/scipy/scipy/pull/13298#discussion_r556778096

Reply:
OK, just ping me when this is ready to review again!